gpiozero/internal_devices.py

```diff
@@ -578,10 +654,19 @@ def end_time(self):
     @property
     def utc(self):
         """
-        If :data:`True`, use a naive UTC time reading for comparison instead of
-        a local timezone reading.
+        If `None` (the default), UTC time will be used for the comparison. 
+        If `False` the local clock time will be use, ignoring the timezone. 
+        (If `True` a naive UTC time will be used - this is not recommended, 
+        see the note above)
         """
         return self._utc
+    
+    @property
+    def aware(self):
```

Comment:
I think `aware` by itself is too ambiguous, perhaps `timezone_aware` would be better?